package.json

```diff
@@ -91,17 +91,17 @@
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "eslint-plugin-tsdoc": "^0.4.0",
     "eslint-plugin-unused-imports": "^4.1.4",
-    "express": "^4.21.2",
+    "express": "^5.1.0",
     "gcp-metadata": "^5.3.0",
     "js-yaml": "^4.1.0",
-    "mocha": "^10.8.2",
+    "mocha": "^11.1.0",
```

Comment:
Mocha 11 set the minimum node to ^18.18.0, seems like it still runs on 16 but it could break on us, do we want to hold off?

Reply:
Good catch -  I don't feel strongly because we're going to drop support for Node16 soon.  Do you have a preference?

Reply:
Doesn't affect me but it could cause churn if a package-lock regen brings in a broken version. Unless there's a feature needed I'd leave it be. FFTR if you still want to keep it.